0.data-download/scripts/nbconverted/data_downloader.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Download Dependency Data
+# 
+# Source: [Cancer Dependency Map resource](https://depmap.org/portal/download/).
+# 
+# - `CRISPR_gene_dependency.csv`: DESCRIBE HERE
+# - `sample_info.csv`: DESCRIBE HERE
+
+# In[2]:
```

Comment:
Looks like you will have to _reset kernel and run all_ again. This should be executed first.